### PR TITLE
[WIP] Add create-dmg

### PR DESCRIPTION
**Pull request by MartinDelille**
….0.3

- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Like I said in the title, this is a work in progress, this is why I don't have checked the items yet.

### CODE REVIEW

File: Formula/create-dmg.rb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Documentation: https://docs.brew.sh/Formula-Cookbook
+#                http://www.rubydoc.info/github/Homebrew/brew/master/Formula
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
```
Comment (SMillerDev) [inline Formula/create-dmg.rb]: This still needs to be removed

File: Formula/create-dmg.rb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+class CreateDmg < Formula
+  desc "A shell script to build fancy DMGs"
+  homepage ""
+  url "https://github.com/andreyvit/create-dmg/releases/tag/v1.0.0.3"
+  sha256 "4557de44fca40e22c0d44b89fcb722d1dc6e6df73049c7756675d4793bf1c9e2"
+  # depends_on "cmake" => :build
```
Comment (SMillerDev) [inline Formula/create-dmg.rb]: Needs to be removed if unused

File: Formula/create-dmg.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+  def install
+    # ENV.deparallelize  # if your formula fails when building in parallel
+    # Remove unrecognized options if warned by configure
```
Comment (SMillerDev) [inline Formula/create-dmg.rb]: Needs to be removed if unused

File: Formula/create-dmg.rb [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    # `test do` will create, run in and delete a temporary directory.
+    #
+    # This test will fail and we won't accept that! For Homebrew/homebrew-core
+    # this will need to be a test that verifies the functionality of the
+    # software. Run the test with `brew test create-dmg`. Options passed
+    # to `brew install` such as `--HEAD` also need to be provided to `brew test`.
+    #
+    # The installed folder is not in the path, so use the entire path to any
+    # executables being tested: `system "#{bin}/program", "do", "something"`.
+    system "false"
```
Comment (SMillerDev) [inline Formula/create-dmg.rb]: Needs an actual test

File: Formula/create-dmg.rb [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+  desc "A shell script to build fancy DMGs"
+  homepage ""
+  url "https://github.com/andreyvit/create-dmg/releases/tag/v1.0.0.3"
+  sha256 "4557de44fca40e22c0d44b89fcb722d1dc6e6df73049c7756675d4793bf1c9e2"
+  # depends_on "cmake" => :build
+
+  def install
+    # ENV.deparallelize  # if your formula fails when building in parallel
+    # Remove unrecognized options if warned by configure
+    system "./configure", "--disable-debug",
```
Comment (SMillerDev) [inline Formula/create-dmg.rb]: Source doesn't seem to have a configure script

File: Formula/create-dmg.rb [hunk 6]
```diff
@@ -0,0 +1,34 @@
+# Documentation: https://docs.brew.sh/Formula-Cookbook
+#                http://www.rubydoc.info/github/Homebrew/brew/master/Formula
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
+class CreateDmg < Formula
+  desc "A shell script to build fancy DMGs"
+  homepage ""
```
Comment (SMillerDev) [inline Formula/create-dmg.rb]: This is needed, but a github repo is also fine.

**Review by SMillerDev — CHANGES_REQUESTED** [review]
This seems like a very rough draft, might want to run it locally a couple of times before submitting a PR?

**Comment by MartinDelille** [comment]
@SMillerDev Yes but I wanted to share it with someone that know `create-dmg` better than me (I'm only a `create-dmg` user).

I was also hopping to receive a little help about creating formula for bash script. Any idea what is going wrong with `bin.install "create-dmg"`? There is not so much help on this topic!

**Pull request closed without merging**